js/__internal/grids/grid_core/views/m_columns_view.ts

```diff
@@ -1086,7 +1086,12 @@ const columnsViewMembers = {
   },
 
   _getCellElementsCore(rowIndex) {
```

Comment:
add please explicit type hints here and in other changed methods

```suggestion
  _getCellElementsCore(rowIndex): dxElementWrapper | undefined {
```